x-pack/legacy/plugins/reporting/server/browsers/chromium/driver/chromium_driver.ts

```diff
@@ -78,6 +84,7 @@ export class HeadlessChromiumDriver {
     logger.info(`opening url ${url}`);
```

Comment:
Please add a `logger.debug` to log the selector string which is passed in. I think we need that now that the pageLoad selector is dynamic. :)

Reply:
The `waitForSelector` method already has a logger.debug statement that outputs the selector. Should I still add one at the start of the openUrl func?

Reply:
oops, my bad - I didn't see that `open` calls `this.waitForSelector` at the end